File: Algorithms/Mathematics/Sum_of_n_natural_numbers/Iterative/python/iterative.py
def sum_n_numbers(n):
    sum = 0
    for i in range(1, n+1):
        sum += i
    return sum

if __name__ == '__main__':
    n = int(input("Enter the number: "))
    print(f"Sum of {n} natural numbers is: {sum_n_numbers(n)}")
    

File: Algorithms/Mathematics/Sum_of_n_natural_numbers/Iterative/cpp/iterative.cpp
#include <iostream>

unsigned long sumNNumbers(unsigned int n) {
    unsigned long sum = 0;
    for(int i=1; i<=n; i++)
        sum += i;
    return sum;
}

int main() {
    std::cout << "Enter the number: ";
    unsigned int n;
    std::cin >> n;
    std::cout << "The sum of " << n << " natural numbers is: " << sumNNumbers(n) << std::endl;
    
    return 0;
}